tls/s2n_server_new_session_ticket.c

```diff
@@ -149,21 +149,6 @@ S2N_RESULT s2n_tls13_server_nst_send(struct s2n_connection *conn, s2n_blocked_st
 
     RESULT_ENSURE(conn->tickets_sent <= conn->tickets_to_send, S2N_ERR_INTEGER_OVERFLOW);
 
-    /* Flush any buffered records to ensure an empty output buffer.
```

Comment:
Do we want to move any part of this comment to post_handshake_write_records?

Reply:
I added a comment to the corresponding flush in post_handshake_write_records, but most of this comment is no longer true.